completions/fish/crio.fish

```diff
@@ -95,13 +96,13 @@ complete -c crio -n '__fish_crio_no_subcommand' -f -l profile -d 'Enable pprof r
 complete -c crio -n '__fish_crio_no_subcommand' -f -l profile-port -r -d 'Port for the pprof profiler'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l read-only -d 'Setup all unprivileged containers to run as read-only. Automatically mounts tmpfs on `/run`, `/tmp` and `/var/tmp`. (default: false)'
 complete -c crio -n '__fish_crio_no_subcommand' -f -l registry -r -d 'Registry to be prepended when pulling unqualified images, can be specified multiple times'
-complete -c crio -n '__fish_crio_no_subcommand' -l root -s r -r -d 'The CRI-O root directory (default: "/var/lib/containers/storage")'
-complete -c crio -n '__fish_crio_no_subcommand' -l runroot -r -d 'The CRI-O state directory (default: "/var/run/containers/storage")'
+complete -c crio -n '__fish_crio_no_subcommand' -l root -s r -r -d 'The CRI-O root directory (default: "/home/mrunalp/.local/share/containers/storage")'
```

Comment:
I'm thinking the mrunalp reference is probably incorrect.

Reply:
Fixed.